README.md

```diff
@@ -55,13 +55,14 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
 
 ```hcl
 module "kops_external_dns" {
-  source       = "git::https://github.com/cloudposse/terraform-aws-kops-external-dns.git?ref=master"
-  namespace    = "cp"
-  stage        = "prod"
-  name         = "external-dns"
-  cluster_name = "us-east-1.cloudposse.com"
-  masters_name = "masters"
-  nodes_name   = "nodes"  
+  source         = "git::https://github.com/cloudposse/terraform-aws-kops-external-dns.git?ref=master"
+  namespace      = "cp"
```

Comment:
use `eg` for example given

Reply:
We should refer to a tag rather than a branch.